src/api/java/openiss/ws/soap/endpoint/ServicePublisher.java

```diff
@@ -37,7 +45,7 @@ public static void main(String[] args) throws Exception {
         OpenISSSOAPServiceImpl service = new OpenISSSOAPServiceImpl();
         Endpoint.publish(url, service);
 
-        if (USE_FILESYSTEM) {
+        if (OpenISSConfig.USE_FAKENECT) {
```

Comment:
Remove that part completely and follow the same logic as in OpenISSRestService.
- inside static initialize the driver and the driver should do kinect.initVideo and kinect.initDepth inside static (need to remove if (ServicePublisher.USE_FREENECT)